test/parsing-test.js

```diff
@@ -21,49 +22,35 @@ describe('Messages parsing', function () {
     })
 
     it('Action getting', function () {
-        assert.equal(messages.getAction(msg), "+todo")
+        assert.equal(formatting.getDataMask(msg, /\+[^*\s]+/), "+todo")
```

Comment:
@weirdGuy I know, just we need tests for that. To be sure that it is actually happening in that way ;)